pages/docs/connect/admin_ui.html

```diff
@@ -10,29 +10,24 @@
 
 `SERVER_IP` is the publicly accessible IP address of any node in the Crate cluster.
 
-![Crate Admin UI]({% media '/media/1512/admin_ui.png' %})
+<img src="/static/images/admin_ui.png">
 
 ## Admin UI Features
 ### Overview Bar and Tab
-![Crate Admin UI Nav Bar]({% media '/media/1512/admin_ui_infobar.png' %})
+<img src="/static/images/status_bar.png">
 
 At the top of the browser window is overview information about the cluster that shows, from left to right:
 
 - The name of the cluster, find out [how to change its value our configuration documentation](https://crate.io/docs/reference/configuration.html#cluster-name)
 - The version of Crate that the cluster is running.
-- The number of nodes in the cluster.
 - The health of the data in the cluster. Green indicates that all data is replicated and available, yellow that there are unreplicated records and red that data is unavailable.
 - Potential configuration problems and recommendations for the cluster.
 - Average cluster load over the past 1, 5 and 15 minutes.
-- Link to Crate documentation.
-- Latest news from Crate.
+- Settings Menu Which allows to update the language settings and provides a link to Crate documentation and the latest news from Crate.
```

Comment:
agree. though would replace "Which allows to" with "that allows you to"

also "to the CrateDB documentation"

"latest CrateDB news"